lib/google_search_data_viewer_web/templates/keyword/index.html.heex

```diff
@@ -4,4 +4,22 @@
   <h3>Select a csv file to upload keywords. </h3>
   <h3>Maximum keywords per file: 1000 </h3>
   <%= render "form.html", Map.put(assigns, :action, Routes.keyword_path(@conn, :upload)) %>
+
+  <h3> Keywords </h3>
```

Comment:
```suggestion
  <h3>Keywords</h3>
```

Reply:
Yes, I still need to find a formatter for these files that also keeps the Emmet shortcuts and highlighting for heex templates, seems you have to have one or the other. Also cleaned up some other inconsistencies, ty ;)